### ISSUE TITLE
error: #include_next is a GCC extension [-Werror]  #include_next <stdio.h>

### ISSUE THREAD

**Issue by E02E02** [Exein-io/exein-openwrt-tensorflow#1]
Hello there,

I got this message related to flatbuffers-v1.11.0 : error: #include_next is a GCC extension [-Werror] 
#include_next <stdio.h>

Cannot build flatbuffers, should I install/add something to the compiler ?

**Comment by Chen-YiDA** [comment]
@E02E02 hello,Are you also trying to install a machine learning framework in openwrt? I'm also studying related ones recently. Can you talk about it?

**Comment by E02E02** [comment]
Yes. It seems, my compiler or is processor, isn't able to perform #include_next. I'm able to fully compile an image with the lib enabled

**Comment by E02E02** [comment]
Yes. It seems, my compiler or is processor, isn't able to perform #include_next. I'm unable to fully compile an image with the lib enabled

**Comment by Chen-YiDA** [comment]
Do you compile tensorflow lite separately or compile it into the openwrt firmware together?

**Comment by E02E02** [comment]
Yes within, I've included : src-git tensorflow https://github.com/Exein-io/exein-openwrt-tensorflow.git then selected in buildroot menu. Openwrt compile when not included.

**Comment by allevi72** [comment]
Hi,
i have the same issue , many openwrt installations (mips, arm ecc.) give me the same error.

**Comment by krsh** [comment]
@E02E02 @allevi72 Which version of OpenWRT and GCC are you using? And which target device?

**Comment by allevi72** [comment]
I used the 8devices devices, kinkan module based on mips 24k (openwrt 10.06.1 toolchain-mipsel_24kc_gcc-5.5.0_musl) and habanero based on arm cortex a7 (openwrt 19.07.05)

**Comment by allevi72** [comment]
18.06.1

**Comment by krsh** [comment]
Use the _glibc_ as C Library: in `Advanced configuration options (for developers) -> Toolchain Options -> C Library implementation`  select __glibc__.

**Comment by E02E02** [comment]
@Chen-YiDA  @allevi72 @krsh  Thx for replying, I've left this issue since last year. I'll do another attempt this week compiling - adding glibc. Feedback soon - Sunshine !
I'll be using  OpenWRT 19.07.7 (previously I was using 18.06) - 8devices Carambola2 board.

**Comment by E02E02** [comment]
Just had an error with xtensor lib, while adding examples - but other that this it went fine - compiler was mips-openwrt-linux-gnu-gcc version 8.3.0 - so thank you especially @krsh, and would be great that it's working with your hardware as well @Chen-YiDA , @allevi72.
